markdown/blockprocessors.py

```diff
@@ -314,6 +308,22 @@ class OListProcessor(BlockProcessor):
     # List of allowed sibling tags.
     SIBLING_TAGS = ['ol', 'ul']
 
+    def __init__(self, parser):
+        BlockProcessor.__init__(self, parser)
+        # Detect an item (``1. item``). ``group(1)`` contains contents of item.
+        self.RE = re.compile(
+                ''.join([r'^[ ]{0,', str(self.tab_length - 1), 
```

Comment:
Wouldn't it make sense to use string formatting here?

```python
r'^[ ]{{0,{0:d}}}\d+\.[ ]+(.*)'.format(self.tab_length - 1)
```

Granted, needing to escape the curly brackets in the regex is slightly annoying. But the old `%`-substitution would work as well:

r'^[ ]{0,%d}\d+\.[ ]+(.*)' % (self.tab_length - 1)

In both cases, the type is defined as an integer (`d`) and converted for us. No need to call `str` explicitly.